packages/hash/shared/src/entityStore.ts

```diff
@@ -76,7 +78,8 @@ export const isDraftBlockEntity = (
 export const getDraftEntityFromEntityId = (
   draft: EntityStore["draft"],
   entityId: string,
-) => Object.values(draft).find((entity) => entity.entityId === entityId);
+): DraftEntity<EntityStoreType> | undefined =>
+  Object.values(draft).find((entity) => entity.entityId === entityId);
```

Comment:
It's the same as what's inferred. I just put that there so that the return type is explicit. 
In recent PRs @kachkaev has pointed out that setting that makes it easy to know what the function returns at a glance and I agree with that. I understand we have a lot of functions that don't have this but we also have functions that do. We can explicitly add the return types for other functions incrementally